src/service/poc_lora.proto

```diff
@@ -145,6 +145,10 @@ message lora_valid_witness_report_v1 {
   uint32 reward_unit = 5;
 }
 
+message invalid_details {
+  oneof data { string denied = 1; }
```

Comment:
if this is now a oneof, should the denylist version be an integer instead of a string since the format is 20230817?

Reply:
I vote leave it as a string since the source is a github tag, which could be something different in the future even if that is unlikely.

Reply:
yeah string is more change of being future proof

Reply:
agreed.. just change it to denylsit_version perhaps?

Reply:
i went with denylist_tag, which is more contextually correct